sherpa/astro/tests/test_fake_pha.py

```diff
@@ -202,24 +215,19 @@ def test_fake_pha_bkg_model():
     assert bkg.counts == pytest.approx(bcounts)
     assert bkg.exposure == pytest.approx(2000)
 
-    # check we've faked counts (the scaling is such that it is
-    # very improbable that this condition will fail)
-    assert (data.counts > counts).all()
-
+    # Apply a number of regression checks to test the output. These
+    # can expect to change if the randomization changes (either
+    # explicitly or implicity). There used to be a number of checks
+    # that compares the simulated data to the input values, but these
+    # could occasionally fail, and so the seed was fixed for these
+    # tests.
+    #
     # For reference the predicted signal is
     #    [200, 400, 400]
     # but, unlike in the test above, this time it's all coming
     # from the background.
     #
-    # What we'd like to say is that the predicted counts are
-    # similar, but this is not easy to do. What we can try
-    # is summing the counts (to average over the randomness)
-    # and then a simple check
-    #
-    assert data.counts.sum() > 500
-    assert data.counts.sum() < 1500
-    # This is more likely to fail by chance, but still very unlikely
-    assert data.counts[1] > 1.5 * data.counts[0]
+    assert data.counts == pytest.approx([186, 411, 405])
```

Comment:
@DougBurke In the comments in line 226 the number of reference counts is different than the one in line 230.
Should these be approximately the same fo the same seed?

Reply:
No. The predicted signal with no noise is 200, 400, 400. The 186, 411, 405 is the value you get with this particular seed.